docs/index.rst

```diff
@@ -476,53 +476,55 @@ compatibility reasons. As `GnuPG documentation
-+--------------+---------------------------------------------------------------+
-| token        | Token serial number                                           |
-+--------------+---------------------------------------------------------------+
-| hash         | Hash algorithm                                                |
-+--------------+---------------------------------------------------------------+
-| curve        | Curve name for elliptic curve cryptography (ECC) keys         |
-+--------------+---------------------------------------------------------------+
-| compliance   | Compliance flags                                              |
-+--------------+---------------------------------------------------------------+
-| updated      | Last updated timestamp                                        |
-+--------------+---------------------------------------------------------------+
-| origin       | Origin of keys                                                |
-+--------------+---------------------------------------------------------------+
-| subkeys      | A list containing [keyid, type] elements for each subkey      |
-+--------------+---------------------------------------------------------------+
-| subkey_info  | A dictionary of subkey information keyed on subkey id         |
-+--------------+---------------------------------------------------------------+
++-------------+--------------------------------------------------------------------------------+
+| dict key    | dict value (all string values)                                                 |
++=============+================================================================================+
+| type        | Type of key                                                                    |
++-------------+--------------------------------------------------------------------------------+
+| trust       | The validity of the key                                                        |
++-------------+--------------------------------------------------------------------------------+
+| length      | The length of the key in bits                                                  |
++-------------+--------------------------------------------------------------------------------+
+| algo        | Public key algorithm                                                           |
++-------------+--------------------------------------------------------------------------------+
+| keyid       | The key ID                                                                     |
++-------------+--------------------------------------------------------------------------------+
+| date        | The creation date of the key in UTC as a Unix timestamp                        |
++-------------+--------------------------------------------------------------------------------+
+| expires     |                                                                                |
```

Comment:
Why is the value column for this row empty?

Reply:
Wait, I think this is a GitHub rendering bug.

Reply:
No you're right, not sure what happened here